wp-content/themes/pub/wporg-learn-2020/archive-course.php

```diff
@@ -21,7 +21,7 @@
 <main id="main" class="site-main">
 	<section>
 		<div class="section-heading section-heading--with-space row align-middle between">
-			<?php the_archive_title( '<h1 class="section-heading_title h2">', '</h1>' ); ?>
+			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
```

Comment:
I've added this class to all the places we use this pattern and have an element on the right, to avoid the same breakage happening.